docker/oso-ops-base/centos7/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM centos:centos7
```

Comment:
Add huge banner saying this is autogenerated and any changes will get stomped on during the next generation.


```
   _____ ______ _   _ ______ _____         _______ ______ _____             _____   ____    _   _  ____ _______   ______ _____ _____ _______ 
  / ____|  ____| \ | |  ____|  __ \     /\|__   __|  ____|  __ \           |  __ \ / __ \  | \ | |/ __ \__   __| |  ____|  __ \_   _|__   __|
 | |  __| |__  |  \| | |__  | |__) |   /  \  | |  | |__  | |  | |  ______  | |  | | |  | | |  \| | |  | | | |    | |__  | |  | || |    | |   
 | | |_ |  __| | . ` |  __| |  _  /   / /\ \ | |  |  __| | |  | | |______| | |  | | |  | | | . ` | |  | | | |    |  __| | |  | || |    | |   
 | |__| | |____| |\  | |____| | \ \  / ____ \| |  | |____| |__| |          | |__| | |__| | | |\  | |__| | | |    | |____| |__| || |_   | |   
  \_____|______|_| \_|______|_|  \_\/_/    \_\_|  |______|_____/           |_____/ \____/  |_| \_|\____/  |_|    |______|_____/_____|  |_|   
                                                                                                                                            
```